lib/locales/es.ts

```diff
@@ -232,6 +233,7 @@ export const locale: Locale = {
     bonaventure_of_bagnoregio_bishop: 'San Buenaventura, obispo y doctor de la Iglesia',
     boniface_of_mainz_bishop: 'San Bonifacio, obispo y mártir',
     bridget_of_sweden_religious: 'Santa Brígida, religiosa',
+    bridget_of_sweden_religious_copatroness_of_europe: 'Santa Brígida, religiosa, patrona de Europa',
```

Comment:
```suggestion
     bridget_of_sweden_religious: 'Santa Brígida, religiosa',  // src: https://liturgiapapal.org/attachments/article/1093/PROPIO%20DE%20LOS%20SANTOS.pdf#page=9
    bridget_of_sweden_religious_copatroness_of_europe: 'Santa Brígida, religiosa, patrona de Europa',  // based on: https://liturgiapapal.org/attachments/article/1093/PROPIO%20DE%20LOS%20SANTOS.pdf#page=9
```

Reply:
sorry about missing these. i'll go put out a new pr if you don't already have one out.

Reply:
resolved in #622